src/player.py

```diff
@@ -4,7 +4,7 @@ class Player(object):
     """Describe the player, his inventory, traits, position and history"""
 
     def __init__(self, state):
-        pass
+        self.state = None
```

Comment:
the constructor has a starting state in parameter...

Reply:
You can do `def __init__(self, state = None)` if you want an optional state